.github/workflows/test.yml

```diff
@@ -141,40 +148,26 @@ jobs:
           build-args: |
             BASE_IMG_TAG=debug
             BUILD_TAGS="netgo,muslc,excludeIncrement"
-      - name: Test e2e and Upgrade
+          cache-from: type=registry,ref=osmosis:debug
+          cache-to: type=inline
```

Comment:
Im going to remove these lines and see if things still work. I remember using cache ended up causing the runners to actually take longer in the past. Will note the diff in time and then re-comment